### DISCUSSION TITLE
Is it illegal to run an auto reload Python script on Github?

### DISCUSSION THREAD

**Discussion by Dantusaikamal** [community/community#22180] [New to GitHub]
<p>I’m learning Python and came across a simple browser auto reload script. Is it safe to use it on Github profile page? I browsed through the rules and didn’t find anything talking about it.</p>
<p>p.s: It’s not a bot, I’m not creating any new accounts. It’s just a simple python code that refreshes my Github browser tab.</p>
<p>~ A github user who doesn’t want to be banned.</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>HI there <a class="mention" href="/dantusaikamal">@Dantusaikamal</a> :wave:</p>
<p>We don’t typically pre-screen content or ideas against our <a href="https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-terms-of-service">Terms of Service</a> or other Site Policy for approval. Instead, we ask that you take the time to familiarize yourself with the policies that are relevant to your use of GitHub and that you especially keep our Acceptable Use Policies in mind with regards to all of your content and conduct on GitHub.</p>
<p>Please note that refreshing a browser is not a prohibition of our Terms of Service, however, we’d like to refer you to our <a href="https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-acceptable-use-policies">Acceptable Use Policies</a>, specifically, section 7 which covers excessive bandwidth usage and may be helpful to keep in mind:<br>
<a href="https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-acceptable-use-policies#7-excessive-bandwidth-use">Excessive bandwidth usage</a></p>

**Comment by Dantusaikamal** [comment]
<p>Thank you, <a class="mention" href="/ernest-phillips">@ernest-phillips</a>. I’ll check out Use policies and bandwidth usage :slight_smile:</p>
